built_value_generator/lib/built_value_generator.dart

```diff
@@ -71,8 +73,7 @@ class BuiltValueGenerator extends Generator {
           '// ignore_for_file: unnecessary_const\n'
           '// ignore_for_file: unnecessary_new\n'
           '// ignore_for_file: test_types_in_equals\n'
-          '\n'
-          '$result';
+          '\n';
```

Comment:
At the end makes sense, but I'd also like to reduce the number of characters by combining them all onto one line, like this:

```
  return '$result\n'
    '// ignore_for_file: '
    'always_put_control_body_on_new_line,'
    'annotate_overrides,'
    'avoid_annotating_with_dynamic,'
    'avoid_as,'
    'avoid_catches_without_on_clauses,'
    'avoid_returning_this,'
    'lines_longer_than_80_chars,'
    'omit_local_variable_types,'
    'prefer_expression_function_bodies,'
    'sort_constructors_first,'
    'test_types_in_equals,'
    'unnecessary_const,'
    'unnecessary_new';
```